sphinx/ext/autosummary/generate.py

```diff
@@ -201,6 +224,21 @@ def get_members(obj, types, include_public=[], imported=True):
                           if x in include_public or not x.startswith('_')]
                 return public, items
 
+            def get_modules(obj, include_public=[]):
```

Comment:
It seems `include_public` is never given. Why is this needed?

Reply:
Same goes for the `get_members`, I just stayed consistent.